project/AppDependencies.scala

```diff
@@ -5,26 +5,27 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
-    "uk.gov.hmrc"          %% "logback-json-logger"           % "5.1.0",
-    "uk.gov.hmrc"          %% "play-conditional-form-mapping" % "1.10.0-play-28",
+    "uk.gov.hmrc"          %% "logback-json-logger"           % "5.2.0",
+    "uk.gov.hmrc"          %% "play-conditional-form-mapping" % "1.11.0-play-28",
     "uk.gov.hmrc"          %% "bootstrap-frontend-play-28"    % "5.24.0",
     "uk.gov.hmrc"          %% "play-frontend-hmrc"            % "3.21.0-play-28",
-    "com.lucidchart"       %% "xtract"                        % "2.2.1"
+    "com.lucidchart"       %% "xtract"                        % "2.0.1"
   )
 
   val test: Seq[ModuleID] = Seq(
-    "org.scalatest"               %% "scalatest"                % "3.2.10",
-    "org.scalatestplus"           %% "mockito-3-2"              % "3.1.2.0",
-    "org.scalatestplus.play"      %% "scalatestplus-play"       % "5.1.0",
-    "org.scalatestplus"           %% "scalatestplus-scalacheck" % "3.1.0.0-RC2",
-    "org.pegdown"                 %  "pegdown"                  % "1.6.0",
-    "org.jsoup"                   %  "jsoup"                    % "1.14.3",
-    "com.typesafe.play"           %% "play-test"                % PlayVersion.current,
-    "org.mockito"                 %  "mockito-core"             % "3.12.4",
-    "org.scalacheck"              %% "scalacheck"               % "1.15.4",
-    "com.github.tomakehurst"      % "wiremock-standalone"       % "2.27.2",
-    "com.vladsch.flexmark"        % "flexmark-all"              % "0.62.2",
-    "uk.gov.hmrc"                 %% "bootstrap-test-play-28"   % "5.14.0"
+    "uk.gov.hmrc.mongo"          %% "hmrc-mongo-test-play-28"  % "0.64.0",
+    "org.scalatest"              %% "scalatest"                % "3.2.12",
+    "org.scalatestplus"          %% "mockito-4-5"              % "3.2.12.0",
+    "org.scalatestplus.play"     %% "scalatestplus-play"       % "5.1.0",
+    "org.scalatestplus"          %% "scalacheck-1-16"          % "3.2.12.0",
+    "org.pegdown"                 % "pegdown"                  % "1.6.0",
+    "org.jsoup"                   % "jsoup"                    % "1.15.1",
+    "com.typesafe.play"          %% "play-test"                % PlayVersion.current,
+    "org.mockito"                 % "mockito-core"             % "4.6.1",
+    "org.scalacheck"             %% "scalacheck"               % "1.16.0",
+    "wolfendale"                 %% "scalacheck-gen-regexp"    % "0.1.1",
```

Comment:
Is this needed? Doesn't appear to have been there before

Reply:
removed